src/httpClient.ts

```diff
@@ -174,14 +169,18 @@ export default class httpClient {
                 sampling_end_date: item.sampling_end_date,
                 sampling_start_date: item.sampling_start_date,
                 seroprevalence: item.serum_pos_prevalence,
+                sensitivity: item.sensitivity,
+                specificity: item.specificity,
                 sex: item.sex,
                 source_id: item.source_id,
                 source_name: item.source_name,
                 source_type: item.source_type,
                 specimen_type: Array.isArray(item.specimen_type) ? item.specimen_type : [item.specimen_type],
                 state: item.state,
                 study_type: item.study_type,
-                test_type: item.study_type,
+                summary: item.summary,
+                test_type: item.test_type,
+                test_manufacturer: item.test_manufacturer,
                 url: item.url
             };
```

Comment:
Is it possible to use object destructuring for this? Check out these [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).

Reply:
Typescript will make this a bit tricky I feel. I think a better long term solution is to find a proper way to map an http response to a typescript object without being so verbose. I will note this in a TODO comment.